perceval/serialization/deserialize.py

```diff
@@ -56,12 +59,15 @@ def matrix_from_file(filepath: str) -> Matrix:
         return deserialize_matrix(f.read())
 
 
-def deserialize_circuit(pb_circ: Union[str, pb.Circuit]) -> Circuit:
+def deserialize_circuit(pb_circ: Union[str, bytes, pb.Circuit]) -> Circuit:
     if not isinstance(pb_circ, pb.Circuit):
         pb_binary_repr = pb_circ
         pb_circ = pb.Circuit()
-        assert pb_binary_repr.startswith(":PCVL:ACircuit:")
-        pb_circ.ParseFromString(b64decode(pb_binary_repr[15:]))
+        if isinstance(pb_binary_repr, bytes):
+            pb_circ.ParseFromString(pb_binary_repr)
+        else:
+            assert pb_binary_repr.startswith(":PCVL:ACircuit:")
```

Comment:
Please put the string in a dedicated variable and use the variable size in the next line.